content/00.build/40.tooling/20.hardhat/00.introduction/02.installation.md

```diff
@@ -0,0 +1,129 @@
+
+- Basic knowledge of deploying smart contracts on Ethereum is beneficial. Refer to
+   the [Contract Deployment](/build/developer-reference/ethereum-differences/contract-deployment) documentation if you need a refresher.
+
+**Wallet and testnet funds:**
+
+- You’ll need a wallet with sufficient Sepolia ETH to deploy contracts. You can obtain Sepolia ETH
+   from [network faucets](https://faucets.chain.link/sepolia) and bridge funds to ZKsync Era using [bridges](https://zksync.io/explore#bridges).
+- Ensure you know how to [export your private key from MetaMask](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
+
+::callout{icon="i-heroicons-light-bulb"}
+If you're migrating an existing hardhat to `zksync-era`, please refer to the [migration guide](/build/tooling/hardhat/guides/migrating-to-zksync) for
+detailed instructions.
+::
+
+### Installation steps
+
+**Install ZKsync CLI:**
+
+- The ZKsync CLI simplifies project setup and local testing. Install it globally with npm or yarn:
+
+::code-group
+
+  ```bash [yarn]
+    yarn global add zksync-cli
+  ```
+
+  ```bash [npm]
+    npm install -g zksync-cli
+  ```
+
+::
+
+**Create a new project:**
+
+- Use the ZKsync CLI to create a new Hardhat project. Choose the template based on your contract language (Solidity or Vyper):
+
+#### Solidity project
+
+```bash
+npx zksync-cli create demo --template hardhat_solidity
+```
+
+#### Vyper project
+
+```sh
+npx zksync-cli create demo --template hardhat_vyper
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+Replace `<project-name>` with your desired project name. This command creates a `demo` folder and clones a Hardhat template project inside it.
+The downloaded project is already configured and contains all the required plugins.
+::
+
+**Configure Hardhat:**
+
+- Open your project's `hardhat.config.ts` file and configure it according to your project needs. This includes setting up ZKsync network
+   endpoints and compiler versions.
+
+**Set up your environment:**
+
+- Rename the `.env.example` file to `.env` and add your wallet’s private key:
+
+     ```text
+     WALLET_PRIVATE_KEY=YourPrivateKeyHere
+     ```
+
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
+For development and testing, it is recommended to use burner wallets. Avoid using real private keys to prevent security risks.
+::
+
+**Install project dependencies:**
+
+- Navigate to your project directory and install dependencies:
+
+::code-group
+
+```bash [yarn]
+  yarn install
+```
+
+```bash [npm]
+  npm install
+```
+
+::
+
+**Compile your contracts:**
+
+- Before deploying, compile your contracts to ensure they are ready for ZKsync:
+
+::code-group
+
+```bash [yarn]
+yarn hardhat compile
+```
+
+```bash [npx]
+npx hardhat compile
+```
+
+::
+
+**Deploy and test:**
+
+- Use Hardhat scripts to deploy your contracts to ZKsync Era. Refer to the [Getting Started](/build/tooling/hardhat/guides/getting-started)
```

Comment:
I am not sure if this page adds much to how to setup a hardhat zksync project. This doesn't describe how to install hardhat-zksync to a project, it explains how to use a project that already has it without explaining what was added to make it a hardhat-zksync project.
We should have this guide have a notice that you can use the existing templates with ZKsync CLI, but the instructions on this page should show how to set up a project without it.